maps/tether/tether-10-colony.dmm

```diff
@@ -25273,7 +25273,7 @@
 /area/unknown/dorm3)
 "WI" = (
 /turf/simulated/floor/wood,
-/area/houseboat)
+/area/space)
```

Comment:
I have a feeling that, given this is a simulated turf, it is not supposed to be area/space.

Reply:
Good point, but it's YOUR ship that it's used on :p

![image](https://user-images.githubusercontent.com/15028025/76627736-f0619d80-6511-11ea-8e52-2aa2fb4d8998.png)

I'll fixy.